holopy/inference/functional_api.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2011-2016, Vinothan N. Manoharan, Thomas G. Dimiduk,
```

Comment:
The code in this file looks great, but can we run it through a pep8 checker? I use pep8online.com

One other regarding pep8 -- per pep8, module names should not have an underscore. Can we rename this from ```functional_api``` to something without an underscore?

Reply:
I use [pycodestyle](https://github.com/PyCQA/pycodestyle) to check pep8 before committing.

Reply:
done. I renamed it to "inference/interface.py" and also renamed scattering/calculations.py to "scattering/interface.py" for consistency. Let me know if you have any concerns about this!